<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_sentiment_classifier_config</class>
 <widget class="QWidget" name="widget_sentiment_classifier_config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>641</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox#groupBox { 
	background-color: rgb(234, 234, 234);
     border: 2px solid gray; 
     border-radius: 3px; 
 } </string>
   </property>
   <property name="title">
    <string>Feature extraction</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>59</y>
      <width>192</width>
      <height>119</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ngram range:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_ngram_range"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="0,0">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>max df:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_max_df"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <widget class="QLabel" name="label_18">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>min df:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_min_df"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <widget class="QLabel" name="label_19">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>max features: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_max_features"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="button_vectorize">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>vectorize data</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>59</y>
      <width>195</width>
      <height>119</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vlayout_checkboxes">
     <item>
      <widget class="QCheckBox" name="check_use_stemming">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>use stemming</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_use_idf">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>use idf</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_binary">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>binary</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_remove_stopwords">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>remove stopwords</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>19</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>vectorizer:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_vectorizer"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>209</y>
      <width>219</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_19" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_20">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>features:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_features">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>641</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox#groupBox_2 { 
	 background-color: rgb(234, 234, 234);
     border: 2px solid gray; 
     border-radius: 3px; 
 } </string>
   </property>
   <property name="title">
    <string>Sentiment classifier</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QWidget" name="layoutWidget_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_24" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_25">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Classifier:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_classifier"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="button_train">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>clusterize data</string>
    </property>
    <property name="text">
     <string>train data</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_22" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_23">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>done in:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_classification_time">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QCheckBox" name="check_perform_lsa">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>perform lsa</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_20">
       <item>
        <widget class="QLabel" name="label_21">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>number of components:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_num_of_components"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_21">
       <item>
        <widget class="QLabel" name="label_22">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>explained variance:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_explained_variance">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>88</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>C:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_regularization"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>210</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Positive sentiment dataset</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_open_positive_sentiment_dataset">
      <property name="toolTip">
       <string>open</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
      <property name="iconSize">
       <size>
        <width>53</width>
        <height>36</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_save_sentiment_classifier_config">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>500</y>
     <width>38</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>216</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Negative sentiment dataset</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_open_negative_sentiment_dataset">
      <property name="toolTip">
       <string>open</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
      <property name="iconSize">
       <size>
        <width>53</width>
        <height>36</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../assets/save.qrc"/>
  <include location="../assets/save_2.qrc"/>
 </resources>
 <connections/>
</ui>
